src/Management/src/OpenTelemetryBase/Trace/ITracingOptions.cs

```diff
@@ -6,24 +6,54 @@ namespace Steeltoe.Management.OpenTelemetry.Trace
 {
     public interface ITracingOptions
     {
+        /// <summary>
+        /// Gets the name of the application
+        /// </summary>
         string Name { get; }
 
+        /// <summary>
+        /// Gets a regex pattern for requests coming into this application that should not be traced
+        /// </summary>
+        /// <remarks>Default value: "/cloudfoundryapplication/.*|.*\\.png|.*\\.css|.*\\.js|.*\\.html|/favicon.ico|/hystrix.stream|.*\\.gif"</remarks>
         string IngressIgnorePattern { get; }
 
+        /// <summary>
+        /// Gets a regex pattern for requests leaving this application that should not be traced
+        /// </summary>
+        /// <remarks>Default value: "/api/v2/spans|/v2/apps/.*/permissions|/eureka/*"</remarks>
         string EgressIgnorePattern { get; }
 
-        int MaxNumberOfAttributes { get; }
```

Comment:
So there is no way to set these anymore - or were they enver used?

Reply:
They _were_ used [here](https://github.com/SteeltoeOSS/Steeltoe/blob/main/src/Management/src/OpenTelemetryBase/Trace/OpenTelemetryTracing.cs#L23), but I don't think they're available options anymore. Searching for them in the OTel repo comes back 0 results of any kind, and the `TracerConfiguration` class is gone